lib/braveAlerter.js

```diff
@@ -304,14 +336,15 @@ class BraveAlerter {
       if (historicAlerts === null) {
         const errorMessage = `Something went wrong in request to ${request.path}. Historic Alerts is null.`
         helpers.logError(errorMessage)
-        response.status(500).send(errorMessage)
+        response.status(500).json(errorMessage)
```

Comment:
Needed to change all these error messages to use `.json` instead of `.send` so that they don't cause errors on the Alert App.